### PR TITLE
Mining page

### PR DESCRIPTION
**Pull request by sohanemon**

### CODE REVIEW

File: piday-web/public/img/mining/receive-success.svg
Comment (Hanggi) [inline piday-web/public/img/mining/receive-success.svg:1]: This is 15.5MB large, use png please

File: piday-web/src/components/BreadCrumb.tsx
Comment (Hanggi) [inline piday-web/src/components/BreadCrumb.tsx:1]: Is there any problem with Joy UI BreadCrumb?

**Review by Hanggi — APPROVED** [review]
I'll approve it, Please continue to pay attention to our convention

File: piday-web/public/img/mining/receive-success.svg
Comment (sohanemon) [reply piday-web/public/img/mining/receive-success.svg:1]: ok

File: piday-web/.vscode/settings.json
[the remark this answers is not in the document]
Comment (sohanemon) [reply piday-web/.vscode/settings.json:1]: I have changed some setting for the workspace. Ill remove this

File: piday-web/src/components/BreadCrumb.tsx
Comment (sohanemon) [reply piday-web/src/components/BreadCrumb.tsx:1]: no, I will add that

**Comment by Hanggi** [comment]
You can merge this pr

**Comment by Hanggi** [comment]
Go for #3

**Pull request closed without merging**